.gitignore

```diff
@@ -6,3 +6,4 @@
 *.test
 *-stamp
 /vendor
+/.vscode
```

Comment:
Please don't include your local editor ignores in the repo. Use [`git config --global core.excludesFile`](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/).

Reply:
```suggestion
```